polyfills/Array/from/tests.js

```diff
@@ -108,11 +108,37 @@ describe('returns an array with', function () {
 			proclaim.deepEqual(Array.from(iterator(3)), [3, 2, 1]);
 		});
 
+		it('can convert from a user-defined iterator using the Symbol.iterator', function () {
```

Comment:
This will needs a skips option for browsers which don't support `Symbol.iterator`.